src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/HunterTalisman.java

```diff
@@ -17,7 +17,12 @@
 	public HunterTalisman(SlimefunItemStack item, ItemStack[] recipe, boolean consumable, boolean cancelEvent, String messageSuffix, int chance) {
 		super(item, recipe, consumable, cancelEvent, messageSuffix, chance);
 	}
-	
+
+	@Override
+	protected boolean areItemHandlersPrivate() {
+		return false;
```

Comment:
This class does not need this method, it doesn't inherit from SimpleSlimefunItem.